src/views/ApartmentApp/index.js

```diff
@@ -27,36 +27,29 @@ const ApartApp = ({ authentication }) => {
   const isOnline = useNetworkStatus();
 
   useEffect(() => {
-    let isSubscribed = true;
-    if (authentication) {
+    const loadPage = async () => {
```

Comment:
and you want the apartapp-develop version here too, right?

Reply:
Actually no, this change was a rather important bug fix that was reviewed and approved by Evan